packages/estatico-boilerplate/src/assets/js/helpers/app.js

```diff
@@ -104,10 +103,11 @@ class App {
       return;
     }
 
-    // jQuery 3 does not support `ready` event in $(document).on() https://jquery.com/upgrade-guide/3.0/#breaking-change-on-quot-ready-quot-fn-removed
-    // But lets sent 'ready' information to modules initialising on that event
-    $(this.initModules.bind(this, { type: 'ready' }));
-    $(document).on(this.initEvents.join(' '), this.initModules.bind(this));
+    const eventDelegate = new EventDelegate(document);
+
+    this.initEvents.forEach((event) => {
+      eventDelegate.on(event, this.initModules.bind(this));
```

Comment:
I think the EventDelegate is not necessary here, right?
Consider using normal vanilla addEventListener if it's not really necessary.